developers/slashing.md

```diff
@@ -0,0 +1,32 @@
+---
+description: This section covers the slashing mechanics for validators in Celestia.
+---
+
+# Slashing on Celestia
+
+Slashing is mechanism employed in PoS blockchains that is used to deter
+and punish malicious behavior. It functions by removing a percentage
+of a validators stake each time they act harmfully towards the network.
+
+Celestia is built with the Cosmos-SDK and implements the module `x/slashing`.
+
+If a Validator gets slashed, delegators bonded to that validator will also
+have the same percentage of their delegated funds slashed.
+
+The following are the conditions for a validator to get slashed:
+
+1. **Downtime**: If a validator is offline for more than 95% of a rolling window
```

Comment:
https://celestiaorg.github.io/celestia-app/specs/params.html could you set the params based on these and link accordingly?

Reply:
Sure! Thanks for reviewing

Reply:
also, use values from the page linked above

Reply:
Please what did you mean by "link accordingly"? There's no way to link each one to the parameter listed in the specs

Reply:
I meant to link to individual params, but I realize that this isn't currently possible. Could you just link to the specs once in the page? thanks!